src/main/scala/ai/privado/exporter/HttpConnectionMetadataExporter.scala

```diff
@@ -41,22 +41,63 @@ class HttpConnectionMetadataExporter(cpg: Cpg, ruleCache: RuleCache) {
   private val STRING_CONTAINS_TWO_SLASH = ".*/.*/.*"
   private val SPRING_APPLICATION_BASE_PATH =
     "(?i)(server[.]servlet[.]context-path|server[.]servlet[.]contextPath)|(spring[.]application[.]name)"
+  private val URL_PATH_WITH_VARIABLE_SYMBOLS                       = "^(?=.*/)(?!.*/$)[${}/\"'a-zA-Z0-9:.,%?_=]+"
+  private val NON_ALPHANUMERIC                                     = "[^a-zA-Z0-9]"
+  private val STRING_WITH_CONSECUTIVE_DOTS_OR_DOT_SLASH_OR_NEWLINE = "(?s).*(\\.\\.|\\./|\n).*"
+  private val ESCAPE_STRING_SLASHES                                = "(\\\")"
+  private val IMPORT_REGEX_WITH_SLASHES                            = "(?s)^(?=.*/)(?!.*/$).*"
 
   private val LAMBDA_SERVERLESS_BASE_PATH = "service"
   private val LAMBDA_SERVERLESS_FILE_NAME = ".*serverless.yml"
 
-  def getEgressUrls = {
+  def getLiteralsFromLanguageFiles: List[String] = {
+    val egressLiterals = cpg
+      .literal(URL_PATH_WITH_VARIABLE_SYMBOLS)
+      .filter(node => node.code.replaceAll(NON_ALPHANUMERIC, "") != "")
+      .filter(!_.code.matches(STRING_WITH_CONSECUTIVE_DOTS_OR_DOT_SLASH_OR_NEWLINE))
+      .inCall
+      .map(node =>
+        node.argument
+          .map(arg => {
+            if node.name == "require" then ""   // skip import statement
```

Comment:
These nodes can be filtered out outside as 
```
inCall
.where(_.nameNot("require"))
.map(node => ....continue
```

Also I belive `import` also needs to be added along with `require`

Reply:
Cool updated accordingly